package.json

```diff
@@ -13,7 +13,8 @@
     "format-check": "prettier --check '{.,pages/**,public/static/docs/**,src/**}/*.{js,md}'",
     "lint-check": "eslint src pages",
     "format-all": "prettier --write '{.,pages/**,public/static/docs/**,src/**}/*.{js,md}'",
-    "format": "prettier --write"
+    "format": "prettier --write",
+    "link-check": "scripts/check-links.sh $(git ls-files '*.md')"
```

Comment:
better to use find? to check files that are not staged yet (or after rename?)?

Reply:
I'd argue that unstaged files should not be checked

Reply:
not with pre-commit, but probably when you do `yarn link-check` you expect it, also that's how other explicit commands work I believe and this one breaks that pattern

Reply:
right fine for consistency. btw I think the `.github/` folder is missing from all these checks...

Reply:
for a good reason :) github templates have their own style because of the way Github renders them. For example, we don't wrap at 80 symbols because of this in the github related files.

I wonder if we should do prettier for the scripts dir though?

Reply:
idk, up to you but I'll add `.github/` to this PR's url checking since I know that's appropriate.

Reply:
yep, URL check are def appropriate! thanks